docs/getting-started/installation-and-upgrade/installation-references/feature-flags.md

```diff
@@ -28,6 +28,7 @@ The following is a list of feature flags available in Rancher. If you've upgrade
 - `rke1-custom-node-cleanup`: Enables cleanup of deleted RKE1 custom nodes. We recommend that you keep this flag enabled, to prevent removed nodes from attempting to rejoin the cluster.
 - `rke2`: Enables provisioning RKE2 clusters. This flag is enabled by default.
 - `token-hashing`: Enables token hashing. Once enabled, existing tokens will be hashed and all new tokens will be hashed automatically with the SHA256 algorithm. Once a token is hashed it can't be undone. This flag can't be disabled after its enabled. See [API Tokens](../../../api/api-tokens.md#token-hashing) for more information.
+- `uiextension`: Allows UI extensions to load without authentication. The `noAuth` field in the CR must be set to `true` for this feature to work. This flag is enabled by default. Enabling or disabling the flag forces Rancher to restart. The first time this flag is set to `true`, it creates a CRD and enables the controllers and endpoints it needs for the feature to work. If set to `false`, it disables the previously mentioned controllers and endpoints. Setting `uiextension` to `false` has no effect on the CRD -- it will not create a CRD if it does not yet exist, nor will it delete the CRD if it already exists. 
```

Comment:
The flag is used to enable/disable the whole UI extension feature, not for enabling/disabling authentication. The `noAuth` field is used for enabling/disabling authentication for that particular UI extension, and it has no impact on the functionality of the `uiextension` flag.

Reply:
@diogoasouza do you think we need to mention `noAuth` here at all, for the purposes of describing the flag?

Reply:
@martyav IMO, we don't need it